galera/files/debian.cnf_slave

```diff
@@ -0,0 +1,13 @@
+{%- from "galera/map.jinja" import slave with context %}
```

Comment:
Don't use two separate files for master + slave in this case, better have one and decide to import slave or master from jinja according to pillar. That way you will avoid duplicating configuration.